.cloud-repo-tools.json

```diff
@@ -0,0 +1,8 @@
+{
+  "product": "redis",
```

Comment:
replace this line with the following:
```
      "_product": "redis",
      "id": "redis",
```
otherwise it won't work as the `"redis"` product is still unknown for the tool.

Reply:
After that, `npm run generate-scaffolding` will work.